mitxpro/management/commands/configure_instance.py

```diff
@@ -0,0 +1,173 @@
+- Creates the OAuth2 application record for edX (optionally with an existing
+  client ID and secret)
+- Create seed data by using seed_data management command
+
+If the --tutor/-T option is passed, the command will use the local.edly.io
+address for links to edX rather than edx.odl.local:18000.
+
+This uses other management commands to complete these tasks. So, if you just
+want to run part of this, use one of these commands:
+- createsuperuser to create the super user
+- configure_wagtail for initial setup of Wagtail assets
+- seed_data to creating seed/dummy courses and programs
+
+There are some steps that this command won't do for you:
+- Completing the integration between MITxPro and devstack - there are still
+  steps that you need to take to finish that process
+
+"""
+
+from django.core.management import BaseCommand, call_command
+from oauth2_provider.models import Application
+
+
+class Command(BaseCommand):
+    """
+    Bootstraps a fresh MITxPro instance.
+    """
+
+    def add_arguments(self, parser):
+        """Parses command line arguments."""
+
+        parser.add_argument(
+            "platform",
+            help="Your platform (none, macos, or linux; defaults to none). None skips OAuth2 record creation.",
+            type=str,
+            choices=["none", "macos", "linux"],
+            nargs="?",
+            const="none",
+        )
+
+        parser.add_argument(
+            "--dont-create-superuser",
+            "-S",
+            help="Don't create a superuser account.",
+            action="store_false",
+            dest="superuser",
+        )
+
+        parser.add_argument(
+            "--edx-oauth-client",
+            help="Use the provided OAuth2 client ID, rather than making a new one.",
+            type=str,
+            nargs="?",
+        )
+
+        parser.add_argument(
+            "--edx-oauth-secret",
+            help="Use the provided OAuth2 client secret, rather than making a new one.",
+            type=str,
+            nargs="?",
+        )
+
+        parser.add_argument(
+            "--gateway",
+            help="Specify the gateway IP. (Required for Linux users.)",
+            type=str,
+            nargs="?",
+        )
+
+        parser.add_argument(
+            "--tutor",
+            "-T",
+            help="Configure for Tutor.",
+            action="store_true",
+            dest="tutor",
+        )
+
+        parser.add_argument(
+            "--tutor-dev",
+            help="Configure for Tutor Dev/Nightly.",
+            action="store_true",
+            dest="tutordev",
+        )
+
+    def determine_edx_hostport(self, *args, **kwargs):  # noqa: ARG002
+        """Returns a tuple of the edX host and port depending on what the user's passed in"""
+
+        if kwargs["tutor"]:
+            return ("local.edly.io", "")
+        elif kwargs["tutordev"]:
+            return ("local.edly.io:8000", ":8000")
+        else:
+            return ("edx.odl.local:18000", ":18000")
```

Comment:
I believe that the default configuration should set the redirect URIs to those of the tutor, as this is the recommended approach for setting up the edX platform.
```suggestion
    def determine_edx_hostport(self, *args, **kwargs):  # noqa: ARG002
        """Returns a tuple of the edX host and port depending on what the user's passed in"""

        if kwargs["devstack"]:
            return ("edx.odl.local:18000", ":18000")
        elif kwargs["tutordev"]:
            return ("local.edly.io:8000", ":8000")
        else:
            return ("local.edly.io", "")
```